index.js

```diff
@@ -1,9 +1,13 @@
 const express = require('express');
 const app = express();
 const PORT = 4000;
+const admins = require('./controllers/admins');
 
 app.get('/', (req, res) => {
     res.send('La Bestia does the best readmes ever');
 })
 
+app.get('/admins', admins.getAdmins)
+app.get('/updateAdmin/:id', admins.updateAdmin)
```

Comment:
Define the entity path and then write the different actions for this entity. Ex: use `/admins/update/:id` instead of `/updateAdmin/:id`

(it has more sense next weeks)

Reply:
Okey, fixed.